<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>110</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StopWatch</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Stopwatch128.png</normaloff>Stopwatch128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_readout">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>190</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>00:00:00.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lap">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>65</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Lap, Lap off, Reset</string>
    </property>
    <property name="text">
     <string>Lap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>65</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start, Stop</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="text">
    <string>Label...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_Start</tabstop>
  <tabstop>pushButton_Lap</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
